manifests/params.pp

```diff
@@ -15,6 +13,17 @@
   $runinterval         = '1800'
   $runmode             = 'service'
 
+  case $::osfamily {
+    'Suse': {
+      $user            = 'root'
```

Comment:
If this is to affect only the ownership of the Puppet dir (e.g. `/etc/puppet`) then you should probably change `$dir_owner` circa line 188 instead.